assets/maps/floor_01/map_9.json

```diff
@@ -46,7 +46,7 @@
          "value":"NEWS"
         }],
  "renderorder":"right-down",
- "tiledversion":"1.10.2",
+ "tiledversion":"1.8.2",
```

Comment:
czemu zmniejszenie wersji tiled?

Reply:
czemu w ogóle zmiana wersji tiled?

Reply:
to jakiś dziwny przypadek, edytowałem sobie trochę mapy i pewnie mam nie aktualną wersję, podbiłem ją